smarts/core/controllers/lane_following_controller.py

```diff
@@ -211,9 +233,16 @@ def perform_lane_following(
             # gain is set to 4.5, the lower the value, the vehicle becomes
```

Comment:
Does the comment here need to be updated? Your call~

Reply:
Thanks, sure.